README.md

```diff
@@ -45,7 +45,7 @@
 
```

Comment:
do we want to change also version in badge on top? https://github.com/KaotoIO/vscode-kaoto/blob/6fc01a4a7049a30072cf7d237865ee77683c8db0/README.md?plain=1#L7